### PR TITLE
Validate request parameters in janus.c with new macro

### PR DESCRIPTION
**Pull request by andreasg123**
To make reviewing easier, I split out the request parameter validation from #526.

### CODE REVIEW

**Comment by lminiero** [comment]
This does help me, thanks! Give me some time to look into this and make some tests to be sure it all still works as expected and I'll get back to you.

File: janus.c [hunk 1]
```diff
@@ -91,11 +91,44 @@ static struct janus_json_parameter incoming_request_parameters[] = {
...
+};
+static struct janus_json_parameter timestamps_parameters[] = {
+	{"timestamps", JSON_INTEGER, JANUS_JSON_PARAM_REQUIRED | JANUS_JSON_PARAM_POSITIVE}
+};
+static struct janus_json_parameter colors_parameters[] = {
+	{"colors", JSON_INTEGER, JANUS_JSON_PARAM_REQUIRED | JANUS_JSON_PARAM_POSITIVE}
+};
+static struct janus_json_parameter mnq_parameters[] = {
+	{"max_nack_queue", JSON_INTEGER, JANUS_JSON_PARAM_REQUIRED | JANUS_JSON_PARAM_POSITIVE}
+};
```
Comment (lminiero) [inline janus.c]: Does `JANUS_JSON_PARAM_POSITIVE` here mean > 0, or >=0? With the latest patch that changes the way `max_nack_queue` works, a value of `0` means "disable it" and so is acceptable.

Comment (lminiero) [inline janus.c]: Sorry for the dumb answer, just checked the code and its >= 0, which works, so please disregard the comment!

**Comment by lminiero** [comment]
This looks fine to me. As soon as you can solve the conflicts (probably caused by some recent merge of mine, sorry about that...) I'll merge, thanks!

**Comment by andreasg123** [comment]
I did a rebase.  The only conflict was in the additional test for `mnq_num` that you added.

**Comment by lminiero** [comment]
Thanks for the quick fix, merging!

**Pull request merged**